sql_files/queries.sql

```diff
@@ -9,3 +9,71 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
 
+BEGIN;
+UPDATE animals SET species = 'digimon';
+WHERE name LIKE '%mon';
+COMMIT;
+SELECT * FROM animals;
```

Comment:
- [ ] According to the project requirements the transaction above should set the species of **all animals** to "unspecified" not "digimon" and also make sure to verify the changes and roll back the transaction and then verify one last time after the rollback.